acir/src/circuit/black_box_functions.rs

```diff
@@ -7,7 +7,7 @@ use strum_macros::EnumIter;
 #[cfg_attr(test, derive(EnumIter))]
 pub enum BlackBoxFunc {
     #[allow(clippy::upper_case_acronyms)]
-    AES,
+    AES128,
```

Comment:
We don't have any specification for this opcode though do we? i.e. we're going to need to provide a message + private key but how are they laid out in the list of function inputs?

Also is this AES encryption or decryption? Should we have two opcodes or have a extra input to choose between? (likely the former)

Reply:
Good points -- I'm gonna block this PR until we can answer those, alternatively we can remove the AES opcode and when we have a proper description for it, we can add it back. What do you think?

Reply:
I think if we don't have a good idea of what it should do then we should remove it. Do we have something in Barretenberg which we can hook this up to?

Reply:
I'm okay with removing it for now, we do have a function in barretenberg but it has not seen much usage so it could have a bunch of bugs